tests/Oryx.Integration.Tests/SqlServerIntegrationTests.cs

```diff
@@ -3,50 +3,176 @@
 // Licensed under the MIT license.
 // --------------------------------------------------------------------------------------------
 
+using System;
+using System.Collections.Generic;
 using System.IO;
 using System.Threading.Tasks;
+using Microsoft.Oryx.Integration.Tests.Fixtures;
+using Microsoft.Oryx.Tests.Common;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Microsoft.Oryx.Integration.Tests
 {
-    //// NOTE:
-    //// Commenting out the tests here as the test fixture is still run and we want to avoid that as stopping the
-    //// SQL Server container on the build agent fails sometimes requiring a reboot of the agent machine.
-
-    //[Trait("category", "db")]
-    //[Trait("db", "sqlserver")]
-    //public class SqlServerIntegrationTests : DatabaseTestsBase, IClassFixture<Fixtures.SqlServerDbContainerFixture>
-    //{
-    //    public SqlServerIntegrationTests(ITestOutputHelper output, Fixtures.SqlServerDbContainerFixture dbFixture)
-    //        : base(output, dbFixture)
-    //    {
-    //    }
-
-    //    [Fact]
-    //    public async Task NodeApp_MicrosoftSqlServerDB()
-    //    {
-    //        await RunTestAsync("nodejs", "10.14", Path.Combine(HostSamplesDir, "nodejs", "node-mssql"));
-    //    }
-
-    //    [Fact]
-    //    public async Task Python37App_MicrosoftSqlServerDB()
-    //    {
-    //        await RunTestAsync("python", "3.7", Path.Combine(HostSamplesDir, "python", "mssqlserver-sample"));
-    //    }
-
-    //    [Theory]
-    //    [InlineData("7.3")]
-    //    [InlineData("7.2")]
-    //    // pdo_sqlsrv only supports PHP >= 7.1
-    //    public async Task PhpApp_UsingPdo(string phpVersion)
-    //    {
-    //        await RunTestAsync(
-    //            "php",
-    //            phpVersion,
-    //            Path.Combine(HostSamplesDir, "php", "sqlsrv-example"),
-    //            8080,
-    //            specifyBindPortFlag: false);
-    //    }
-    //}
+    [Trait("category", "db")]
+    [Trait("db", "sqlserver")]
+    public class SqlServerIntegrationTests : PlatformEndToEndTestsBase
+    {
+        private const string DbServerHostnameEnvVarName = "SQLSERVER_DATABASE_HOST";
```

Comment:
I am going to set these values on the build definitions to avoid storing sensitive data

Reply:
can you store them in a Build Variable Group? this way, it can be shared among different build definitions, can't it?

Reply:
I haven't checked that yet, but will do now.

Reply:
Seems like Elena had created a variable group called 'Oryx' for our team already, we just need to sync it up with these new variables. Working on it...